<launch>
	<node name="motor_controller" pkg="kocheng" output="screen" type="motor_controller"/>
	<node name="node_controller" pkg="kocheng" output="screen" type="node_controller" />
	<node name="remote_monitor" pkg="kocheng" output="screen" type="remote_monitor" />
	<node name="debug_makara" pkg="kocheng" output="screen" type="debug_makara" />
</launch>
